core/Cargo.toml

```
@@ -27,6 +27,9 @@ bs58 = { version = "0.4" , optional = true }
 base64 = { version = "0.22", optional = true }
 serde_json = { version = "1.0", optional = true }
 hex = { version = "0.4" , optional = true }
+hkdf = "0.12.4"
+sha2 = "0.10.8"
```

```suggestion
hkdf = "0.12"
sha2 = "0.10"
```

Instead of specifying a particular subversion, I think we can change that to the major version only, as with other dependencies in the Cargo.toml